packages/cli/test/cli.test.js

```diff
@@ -187,16 +164,15 @@ describe('Lighthouse CI CLI', () => {
 
       const stdoutClean = stdout.replace(/sqlDatabasePath=.*?"/, 'sqlDatabasePath=<file>"');
       expect(stdoutClean).toMatchInlineSnapshot(`
-        "Started a web server with \\"yarn start server -p=XXXX --storage.sqlDatabasePath=<file>\\"...
```

Comment:
revert these few lines? :)

Reply:
Prettier not helping me after moving tests around 😅

Reply:
Done!